spec/helpers/single_search_helper_spec.rb

```diff
@@ -100,7 +100,7 @@
       end
 
       it "returns a link to the catalogue search results page" do
-        expect(helper.bento_all_results_link("catalogue")).to eq("http://test.host/catalog?search_field=all_fields&q=hello")
+        expect(helper.bento_all_results_link("catalogue")).to eq("http://test.host/catalog?search_field=all_fields&q=hello&search_type=simple")
```

Comment:
Is there a way to always append this unless it's an advanced search? I guarantee this will be forgotten when someone (i.e. me) makes a change in the future.

Reply:
I couldn't find a single place where I could inject this, I was going to use the search_field=advanced but it seemed safer to set a separate field from wherever the search was invoked. I can have another look though - the search is a bit like whack-a-mole, change one thing and break another

Reply:
I didn't realise the blacklight_params hash was in scope in the builder. I have set it in the builder and remobved it from everywhere else which is much nicer. Thank you for making me take another look.